MediaBrowser.Providers/Movies/MovieDbProvider.cs

```diff
@@ -287,7 +287,7 @@ public static string NormalizeLanguage(string language)
             if (!string.IsNullOrEmpty(language))
             {
                 // They require this to be uppercase
-                // https://emby.media/community/index.php?/topic/32454-fr-follow-tmdbs-new-language-api-update/?p=311148
+                // https://jellyfin.media/community/index.php?/topic/32454-fr-follow-tmdbs-new-language-api-update/?p=311148
```

Comment:
This shouldn't be changed, the link is there to inform devs why it's done this way. While the new link just points to the organization. A better change would be writing out the info instead of linking to a forum post.

Reply:
Sorted with: https://github.com/jellyfin/jellyfin/pull/332/commits/d9d3865bc1f25e5e425c13d25839dd1a1ab89204